docusaurus/docs/reactnative/guides/message_input_customization.mdx

```diff
@@ -489,3 +489,53 @@ const CustomAttachButton = () => {
   <Channel AttachButton={CustomAttachButton} />
 </ActionSheetProvider>;
 ```
+
+## Creating a Custom Editing state Header in MessageInput
+
+To make a custom editing state header, you can use the [Channel](../core-components/channel.mdx) component with a custom Editing state header component and pass it inside [InputEditingStateHeader](../core-components/channel.mdx#inputeditingstateheader) prop.
+
+```tsx
+import { Text } from 'react-native';
+import { Channel } from 'stream-chat-react-native';
+
+const CustomInputEditingStateHeader = () => <Text style={{ textAlign: 'center' }}>Edit Message</Text>;
```

Comment:
Could you also add a close button in this example, and show how they can attach clearEditingState from context to that button.

Reply:
Done @vishalnarkhede